### PR TITLE
add raleigh-durham chapter

### PR DESCRIPTION
**Pull request by jbelmont**

### CODE REVIEW

File: source/chapters.yml [hunk 1]
```diff
@@ -157,30 +157,35 @@
...
 - name: san-diego
   title: San Diego
   description: The San Diego chapter of Papers We Love
-  url: "https://www.meetup.com/Papers-We-Love-San-Diego/"
+  url: https://www.meetup.com/Papers-We-Love-San-Diego/
   meetup_url: Papers-We-Love-San-Diego
+- name: raleigh-durham
+  title: Raleigh-Durham
+  description: The Raleigh-Durham chapter of Papers We Love
+  url: /chapters/raleigh-durham
```
Comment (DarrenN) [inline source/chapters.yml]: `/chapter/raleigh-durham`

File: source/partials/chapters/_raleigh-durham.erb.markdown [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+The Raleigh-Durham chapter of Papers We Love
+
+Papers We Love has a **[Code of Conduct](https://github.com/papers-we-love/papers-we-love/blob/master/CODE_OF_CONDUCT.md)**. Please contact one of the Meetup's organizers if anyone is not following it. Be good to each other and to the PWL community!
+
+## Chapter details
+
+**Sign-up:** Please RSVP for meetings via <a href="http://www.meetup.com/Papers-We-Love-Raleigh-Durham">Meetup.com</a>
+
+**Twitter:** <a href="https://twitter.com/CHAPTER">@CHAPTER</a>
```
Comment (DarrenN) [inline source/partials/chapters/_raleigh-durham.erb.markdown]: Does your chapter have a Twitter account?

File: source/partials/chapters/_raleigh-durham.erb.markdown [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+**Organizers:**
+
```
Comment (DarrenN) [inline source/partials/chapters/_raleigh-durham.erb.markdown]: Can you add your contact info here? (Twitter / Github / etc.)

**Comment by jbelmont** [comment]
@DarrenN I updated with your recommended changes how does it look?

**Comment by DarrenN** [comment]
@jbelmont looks great, thanks for the updates!

**Pull request merged**